crypto/vrf/vrf_libsodium.go

```diff
@@ -18,6 +18,8 @@ func init() {
 	defaultVrf = newVrfEd25519libsodium()
 }
 
+const LibsodiumProofLength = 80
```

Comment:
The definitive size of the VRF Proof of libsodium is defined in `crypto/vrf/internal/vrf/vrf.go`.